src/cells/model.ts

```diff
@@ -357,8 +357,8 @@ class BaseCellModel implements IBaseCellModel {
   type: CellType;
 
   private _input: IInputAreaModel = null;
-  private _tags = '';
-  private _name = '';
+  private _tags = '{}';
+  private _name = '{}';
```

Comment:
It can't be empty, since that causes the `JSON.parse()` to fail, agreed that tags should be `[]` though.